### PR TITLE
Guide notebooks use the python API

### PR DESCRIPTION
**Pull request by ohinds**
Remove old CLI-based notebooks and rework API notebooks to be more clear.

TODO: add an API-based notebook for brain generation.

### CODE REVIEW

File: docs/nobrainer-guides/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+This set of guides demonstrates basic usage of the `nobrainer` python API to train and use neural network models for neuroimaging. The `nobrainer` project was developed under the support of NIH RF1MH121885 and R01EB020470. It is distributed under the Apache 2.0 license.
+
+## Accessing the guides
+
+These guides are available in both jupyter notebook versions and as standalone python scripts.
+
+### Jupyter notebooks
+
+See the [notebooks](notebooks) directory for the guides in jupyter notebook format. These notebooks can be run for free on Google Colab, accessed [here](https://colab.research.google.com/github/neuronets/nobrainer-book/blob/ohinds-guide-api).
```
Comment (satra) [inline docs/nobrainer-guides/README.md]: this should be updated before this is merged.

**Review by satra — COMMENTED** [review]
this should be merged when more of the guides are in place. 

i would say these at least:
- warm start (i.e. fine tuning/transfer learning) 
- multi-GPU
- augmentation

there are two other pieces i would like to see:
- bayesian models
- synth-based training @hvgazula 

label-free options
- generation (i know this is ready, so this could be added earlier)
- autoencoders
- latent diffusion models

File: docs/nobrainer-guides/README.md (hunk 1 above)
Comment (ohinds) [reply docs/nobrainer-guides/README.md]: :+1:

**Comment by ohinds** [comment]
> this should be merged when more of the guides are in place.
> 
> i would say these at least:
> 
> * warm start (i.e. fine tuning/transfer learning)
> * multi-GPU
> * augmentation
> 
> there are two other pieces i would like to see:
> 
> * bayesian models
> * synth-based training @hvgazula
> 
> label-free options
> 
> * generation (i know this is ready, so this could be added earlier)
> * autoencoders
> * latent diffusion models

I agree that those should be added in quick succession after this, and if you want a monolithic PR that's fine too, I can change this to a draft if you like. I'm more interested in feedback on the general approach before I move on to the other guides.

But also this is blocking PRs in the `nobrainer` repo to remove the guides and implement CI. I'd propose that I file issues to add each of these guides, merge this, then merge the unblocked things in `nobrainer`, then have a separate PR for successive guides?

**Comment by satra** [comment]
if you mean about writing in python and then converting to ipynb, then that's fine. i tend to do it with markdown and convert to ipynb and python, because it's easier to write the comment cells in markdown. or if i'm doing dev in jupyter then just in notebook format and have it auto write to python/markdown. 

it's just that this initial merge completely removes those other notebooks. also one of the reason to move the guides to this repo is to create a structured and executable documentation for nobrainer as a whole (nobrainer library, tutorials, nobrainer-zoo, etc.,.) as an example see this jupyter book: https://alleninstitute.github.io/openscope_databook/intro.html

**Comment by ohinds** [comment]
> if you mean about writing in python and then converting to ipynb, then that's fine. i tend to do it with markdown and convert to ipynb and python, because it's easier to write the comment cells in markdown. or if i'm doing dev in jupyter then just in notebook format and have it auto write to python/markdown.
> 
> it's just that this initial merge completely removes those other notebooks. also one of the reason to move the guides to this repo is to create a structured and executable documentation for nobrainer as a whole (nobrainer library, tutorials, nobrainer-zoo, etc.,.) as an example see this jupyter book: https://alleninstitute.github.io/openscope_databook/intro.html

I see. I thought we were replacing the cli-based notebooks with the API-based ones. But you are saying they should both be in the guide?

**Pull request merged**